platform/iphone/export/export.cpp

```diff
@@ -279,12 +279,14 @@ Error EditorExportPlatformIOS::export_project(const Ref<EditorExportPreset> &p_p
 			f->close();
 			memdelete(f);
 
-#ifdef OSX_ENABLED
 			if (is_execute) {
+#ifdef OSX_ENABLED
 				// we need execute rights on this file
 				chmod(file.utf8().get_data(), 0755);
-			}
+#else
+				ERR_PRINT("Binary can't be made executable on this platform.");
```

Comment:
`WARN_PRINT` would be better IMO, an error sounds like an editor bug, which it is not.

Also as I mentioned, I think `UNIX_ENABLED` would be worth trying as I don't see why the `chmod` wouldn't work from Linux. It's just Windows which has no clue about file permissions.

Reply:
I'll make these changes